editor/src/components/navigator/navigator-item/navigator-condensed-entry.tsx

```diff
@@ -87,6 +88,14 @@ export const CondensedEntryItemWrapper = React.memo(
       )
     }, [selectedViews, props.navigatorRow])
 
+    const { isTopOfSelection, isBottomOfSelection } = useNavigatorSelectionBoundsForEntry(
+      props.navigatorRow.entries[0],
+      rowRootSelected,
+      0,
+    )
+
+    const borderRadius = 5
```

Comment:
I'd put this into a const, this number also shows up in other places too

Reply:
https://github.com/concrete-utopia/utopia/pull/5928/commits/c3c0db05158427cb6341959753a1bb13234d1a16